pymemcache/client/rendezvous.py

```diff
@@ -36,7 +36,7 @@ def get_node(self, key):
 
         for node in self.nodes:
             score = self.hash_function(
-                "%s-%s" % (str(node), str(key)))
+                "%s-%s" % (node, key))
```

Comment:
Oh, you're right actually, the casting happens automatically with the string substitution, I missed that. Thanks